lib/dai-plugin-mcd/src/constants.js

```diff
@@ -1,3 +1,4 @@
 export const ServiceRoles = {
-  CDP_MANAGER: 'cdpManager'
+  CDP_MANAGER: 'cdpManager',
+  CDP_TYPE: 'collateral'
```

Comment:
let's keep these constants the same as their string values (just converted into all caps with underscores) rather than having them be different